files/image_config/corefile_uploader/core_analyzer.rc.json

```diff
@@ -0,0 +1,17 @@
+{
+    "local_work": {
+        "core_upload": "/tmp/core_upload/"
+    },
+    "azure_sonic_core_storage": {
+        "account_name": "corefilecollection",
```

Comment:
can you use template for both name and share_name?

Reply:
You can use /usr/bin/update_json.py to update any of the entries.